src/js/chat.js

```diff
@@ -228,8 +244,15 @@ function formatTime( string ) {
  *  Display a message in the chat window
  *  @param raw_msg: The contents of the privmsg to be posted
  *  @param isHistory: If true, it should be pushed at the top, as it is an older message (and may be coming in late)
+ *  
  */
-function postMessage( raw_msg, isHistory ) {
+function postMessage(raw_msg, isHistory ) {
+    if (!isHistory && !connected) {
+        toBePosted.push(raw_msg);
+        return;
+    }
+    postedMessages.push(raw_msg.trim());
+    //console.log("after while");
```

Comment:
Remove this

Reply:
Why? without it I can't know which messeges from history did I already post and which not

Reply:
I was referring to the commented out console statement